### PR TITLE
Added option to set Http Client timeout

### PR DESCRIPTION
**Pull request by alesak**

### CODE REVIEW

**Comment by kreznik-augury** [comment]
Hey @alesak, could you kindly add a test for the new feature?

File: src/Client.php [hunk 1]
```diff
@@ -371,4 +380,22 @@ protected function paramsToQueryString(array $params = [])
         }
         return implode('&', $query_string);
     }
+
+    /**
+     * @return float|int
+     */
+    public function getConnectTimeout(): float
```
Comment (Aternus) [inline src/Client.php]: It looks like in the annotation you've specified `float|int` but in the return type there is only `float`.
In addition, currently, the project is not using built-in PHP types. It might be good to have PHP types but it should be done as a separate PR.

**Review by Aternus — CHANGES_REQUESTED** [review]

**Review by Aternus — APPROVED** [review]
